collect_app/src/main/java/org/odk/collect/android/activities/FormHierarchyActivity.java

```diff
@@ -361,7 +360,7 @@ public void refreshView() {
             // set the controller back to the current index in case the user hits 'back'
             formController.jumpToIndex(currentIndex);
         } catch (Exception e) {
-            Log.e(t, e.getMessage(), e);
+            Timber.e(e.getMessage(), e);
```

Comment:
Make sure to read the `Timber` docs carefully. This should just be `Timber.e(e)`!